fastlane/Fastfile

```diff
@@ -88,6 +88,11 @@ end
 
 lane :merge_release do |options|
   merge_release_to_main(author: options[:author])
+  sh('gh workflow run release-publish.yml --ref main')
```

Comment:
Did you verify that it works on CI?

Reply:
https://github.com/GetStream/stream-chat-swift/actions/runs/11126371804/job/30916141810 👍